modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotMessageCounter.java

```diff
@@ -102,11 +112,10 @@ public void run() {
                         } catch (ConnectionException e) {
                             log.error("Error while sending slot deletion safe zone update", e);
                         }
-
                     }
                 }
             }
-        }, 4000, 3000);
+        }, SLOT_SUBMIT_TIMEOUT, SLOT_SUBMIT_TASK_EXEC_PERIOD);
```

Comment:
We need to ask @hastef88 on this. IMO this depends on the publisher rate as well. in a slow publishing rate this task comes into play. Then this time depends on the maximum message delay we can have and the number of messages for a submit slot event.